fvm/Cargo.toml

```diff
@@ -45,6 +45,7 @@ rand = "0.8.5"
 
 [dev-dependencies]
 pretty_assertions = "1.2.1"
+fvm = { path = ".", features = ["testing"] }
```

Comment:
The version-2 feature resolver will prevent this feature from infecting non-dev builds.